README.md

```diff
@@ -217,7 +217,8 @@ build GoAccess from source.
 
 A Docker image has been updated, capable of directing output from an access log. If you only want to output a report, you can pipe a log from the external environment to a Docker-based process:
 
-    cat access.log | docker run --rm -i -e LANG=$LANG allinurl/goaccess -a -o report.html --log-format COMBINED -
+    touch report.html
+    cat access.log | docker run --rm -i -v ./report.html:/report.html -e LANG=$LANG allinurl/goaccess -a -o report.html --log-format COMBINED -
```

Comment:
```suggestion
    touch report.html
    cat access.log | docker run --rm -i -v ./report.html:/report.html -e LANG=$LANG allinurl/goaccess -a -o report.html --log-format COMBINED -
    
    OR
    
    cat access.log | docker run --rm -i -e LANG=$LANG allinurl/goaccess -a -o report.html --log-format COMBINED -
```

Reply:
I think I'm missing something. Won't the second command

1. start the container
2. read and transform the logs
3. write the logs to an internal report
4. shutdown and remove the container -> the report is lost

?

Reply:
My bad, I mistakenly thought it was a real-time process.